handout/forms/template-driven/validating_forms.md

```diff
@@ -15,7 +15,7 @@ Using the template-driven approach, form validation is a matter of following HTM
 <input type="text" ... pattern=".{3,8}" required>
 
 <!-- alphanumeric field of specific length -->
-<input type="text" pattern="[A-Za-z0-9]{0,5}">
+<input type="text" ... pattern="[A-Za-z0-9]{0,5}">
```

Comment:
This is a bit contentious.  We had `...` like this at one point but some readers took the `...` too literally so we removed it.  I think it's up for discussion if you have reasons.